app/assets/stylesheets/application.scss

```diff
@@ -28,3 +35,71 @@
     margin: 0;
   }
 }
+
+/* Move down content because we have a fixed navbar that is 50px tall */
+body {
+  padding-top: 50px;
+}
+
+/*
+ * Sidebar
+ */
+
+/* Hide for mobile, show later */
+.sidebar {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .sidebar {
+    position: fixed;
```

Comment:
Properties should be ordered background-color, border-right, bottom, display, left, overflow-x, overflow-y, padding, position, top, z-index